mycore-oai/src/main/resources/META-INF/resources/oai/oai2.xsl

```diff
@@ -512,7 +512,7 @@ p.intro {
   <xsl:apply-templates select="oai:setSpec" />
   </table>
   <xsl:if test="@status='deleted'">
-    <p>This record has been deleted.</p>
```

Comment:
oai2.xsl is the nearly unchanged file as it was provided by eprints.
I am not sure, if we should start modifying texts in it.

Reply:
There have already been multiple modifications to the file, so this shouldn't be any additional issue. But it would answering potential questions preemptively.

Has there ever been an effort to keep the version of this file used by MyCoRe in sync with the version used by EPrints and to backport changes made by the EPrints community?